app/src/raw_data_store.rs

```diff
@@ -0,0 +1,51 @@
+use alloc::collections::BTreeMap;
+use alloc::vec::Vec;
+
+/**
+ * All models must be defined in this list, which is filled at compile time.
+ */
+const RAW_DATA_LIST: [(&str, &'static [u8]); 4] = [
+    ("Suzanne", include_bytes!("../data/Suz.obj")),
+    ("Suzanne_mat", include_bytes!("../data/Suz.mtl")),
+    ("Triangle", include_bytes!("../data/Tri.obj")),
+    ("Triangle_mat", include_bytes!("../data/Tri.mtl")),
+];
+
+/**
+ * Data structure for the raw data store
+ */
+pub struct RawDataStore {
+    raw_data_map: BTreeMap<&'static str, &'static [u8]>,
+}
+
+/**
+ * Implementation of the raw data store: allows for fast searching.
+ */
```

Comment:
Good idea, and good simple implementation 👍

Reply:
Thanks :)